springfield/firefox/templates/firefox/browsers/compare/base-article.html

```diff
@@ -13,7 +13,7 @@
   {{ css_bundle('firefox-compare-article') }}
 {% endblock page_css %}
 
-{% set utm_params = '?utm_source=www.firefox.com&utm_medium=referral&utm_campaign=browser-comparison' %}
+{% set referrals = '?utm_source=www.firefox.com&utm_medium=referral&utm_campaign=browser-comparison' %}
```

Comment:
I'm not overly opinionated on this beyond wanting more consistency. Happy to adjust to a different standard as needed (i.e. always called "utm_params" and never including question mark)

Currently, we have some "referrals" and some "utm_params" or "_params" or "utms". Sometimes they start with a question mark, sometimes not.